website/content/docs/auth/kerberos.mdx

```diff
@@ -61,7 +61,7 @@ $ vault auth enable \
     kerberos
 ```
 
-- Create a `keytab` for the Kerberos plugin:
+- Create a `keytab` for the Kerberos plugin (this keytab is used by the Vault server itself, another keytab should be generated for login purposes):
```

Comment:
```suggestion
- Create a `keytab` for the Kerberos plugin. Since this keytab is used by the Vault server, another keytab should be generated for login purposes. 
```